docs/semgrep-app/integrations.md

```diff
@@ -8,12 +8,97 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Integrations
 
+Semgrep App contains third party integrations to allow users to add data from Semgrep to other tools that are part of their workflows.
+
+Currently, Semgrep App integrates with the following tools:
+
+| Tool | Tier availability |
+| ---- | ---------------- |
+| Slack | Community (Free) |
+| Email | Community (Free) |
+| Jira | Team |
+| Amazon S3 | Team |
+| Webhook | Team |
+
+## Where are integrations found?
+
+After logging into your Semgrep App account, you can find **Integrations** as a tab within **Settings.**
+
+![Screenshot of Semgrep's "Create New Integration Channel" menu](../img/integration-firstview.png)<br />
+
+View your saved integration channels by clicking on the **gear** icon within the **Rule board**. Click on the toggles to turn notifications on or off for each channel.
+
+![Screenshot of the Semgrep rule board with integrations](../img/integration-ruleboard.png)<br />
+
+## Slack
+
+Integrating with Slack allows Semgrep to post messages after each scan about new findings to a channel on your Slack workspace.
+
+To set it up, you'll need to create an app within Slack and enable that app's **Incoming Webhooks** feature. By creating a webhook, you'll have a URL which serves as Semgrep's endpoint to your Slack workplace. The guide below will walk you through those steps.
+
+1. Make sure that you're logged into your Slack account. You can enter your login details [here.](https://slack.com/workspace-signin)
```

Comment:
```suggestion
1. Log in to your Slack account. Enter your login details in [Slack workspace](https://slack.com/workspace-signin).
```

Reply:
Just heads up :-), be sure to:

1. Use something concrete for the link's text. Do not use click here, click this. Say what a reader can expect inside of the link.
2. Do not include full stops in links.
3. Make steps imperatives.

Reply:
Agree on all points. I'd like to add though, that in some cases "Make sure you are in [x state]" is an appropriate phrase, as seen in this [django tutorial](https://github.com/microsoft/vscode-docs/blob/main/docs/python/tutorial-django.md). Have committed your suggestion!